CreeDictionary/search_quality/sample.csv

```diff
@@ -1,70 +1,384 @@
-i see you,kiwâpamitin,,,,,,
-he sees him,wâpamêw,,,,,,
-she sees him,wâpamêw,,,,,,
-starblanket,atâhkakohp,acâhkosa kâ-otakohpit,,,,,
-star blanket,atâhkakohp,acâhkosa kâ-otakohpit,,,,,
-being taught,kiskinwahamâkosiw,,,,,,
-they see us,niwâpamikonânak,kiwâpamikonawak,,,,,
+Query,Nêhiyawêwin 1,Nêhiyawêwin 2,Nêhiyawêwin 3,,Notes
+see,wâpamêw,wâpiw,wâpahtam,,"Ignore ""\[see"""
+eat,mîcisow,mîciw,mowêw,,
+fire,iskotêw,,,,
+happy birthday,miyo-tipiska,,,,
+thank you,ay-hay,hay-hay,kinanâskomitin,,
+welcome,tatawâw,,,,
+'horse',misatim,mistatim,,,
+"""horse""",misatim,mistatim,,,
+beaverberry,amiskomin,amiskominak,,,
+beaverberries,amiskomin,amiskominak,,,
+beaver-berry,amiskomin,amiskominak,,,
+beaver-berries,amiskomin,amiskominak,,,
+beaver berry,amiskomin,amiskominak,,,
+beaver berries,amiskomin,amiskominak,,,
+berries,mînisa,mînis,,,
+berry,mînis,mînisa,,,
+berri,mînis,mînisa,,,
+stove pipe,okohtaskwahikan,,,,
+stove-pipe,okohtaskwahikan,,,,
+stovepipe,okohtaskwahikan,,,,
+in the area,cîkâhtaw,,,,
+bread,pahkwêsikan,,,,
+yellow hat,osâwastotin,,,,
+sleep,nipâw,,,,
+nîminâniwan,nîminâniwan,nîmiw,,,
+I love you,kisâkihitin,,,,
+usual,mâna,,,,
+no,namoya,,,,
+this,ôma,awa,ôhi,,
+that,anima,,,,
+good morning,miyo-kîkisêpâyâw,,,,
+me,niya,,,,
+good bye,ekosi mâka ,,,,
+goodbye,ekosi mâka ,,,,
+work together,mâmawi-isîhcikêwin,,,,
+working together,mâmawi-isîhcikêwin,,,,
+work togeth,mâmawi-isîhcikêwin,,,,
+Cree,nêhiyaw,nêhiyawêwin,,,
+Edmonton,amiskwaciy-wâskahikan,,,,
+edmonton,amiskwaciy-wâskahikan,,,,
+hill,waciy,ispatinaw,,,
+beaver hill,natimîwiyiniwak,amiskwacîwiyiniwak,,,
+beaver hills,natimîwiyiniwak,amiskwacîwiyiniwak,,,
+Calgary,otôskwanihk,,,,
+calgary,otôskwanihk,,,,
+car,sêhkêpayîs,,,,
+water,nipiy,,,,
+telling a story,âcimêw,âcimow,,,
+on water,nipîhk,,,,"aspirational search, as result is based on inflections, not a direct definition entry"
+hello,tânisi,,,,
+difficult thing,âyiman,,,,
+many snakes,kinêpikoskâw,,,,
+there are many snakes,kinêpikoskâw,,,,
+dances,nîmiw,nîmihitow,,,
+dancing time,nîminâniwan,nîmiw,,,
+go to bed,mâci-nipa,,,,
+Merry Christmas,miyo-manitowi-kîsikânisik,,,,
+merry christmas,miyo-manitowi-kîsikânisik,,,,
+i see you,kiwâpamitin,,,,
+he sees him,wâpamêw,,,,
+she sees him,wâpamêw,,,,
+starblanket,atâhkakohp,acâhkosa kâ-otakohpit,,,
+star blanket,atâhkakohp,acâhkosa kâ-otakohpit,,,
+being taught,kiskinwahamâkosiw,,,,
+they see us,niwâpamikonânak,kiwâpamikonawak,,,
+gym,mêtawêwikamik,,,,This item and below are from NS152
+lake,sâkahikan,,,,
+juice,sîwâpoy,,,,
+hospital,âhkosîwikamik,,,,
+sock,asikan,,,,
+pail,askihk,,,,
+potato,askipwâwi,,,,
+dog,atim,nitêm,,,very irregular inflections
+doll,awâsisihkân,,,,
+tobacco,ciscêmâs,cistêmâw,,,
+fork,cîscahikanis,,,,
+girl,iskwêsis,,,,
+woman,iskwêw,,,,
+fish,kinosêw,,,,
+bacon,kohkôsiwiyin,,,,
+corn,mahtâmin,,,,
+pencil,masinahikanâhtik,,,,
+slate,masinahikanâpisk,,,,
+pen,masinahikanâpiskos,,,,
+tea,maskihkîwâpoy,,,,
+bear,maskwa,,,,
```

Comment:
bear: maskwa is the best top term, but I would love to see some other words for black bear or grizzly as well (polar bears are perhaps less relevant to Plains Cree, but there is a word). There are also some avoidance terms (words for bear are often taboo), but I'm not sure people need those at the very top)